go.mod

```diff
@@ -127,6 +128,7 @@ require (
 	github.com/smallstep/pkcs7 v0.1.1 // indirect
 	github.com/stefanberger/go-pkcs11uri v0.0.0-20230803200340-78284954bff6 // indirect
 	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635 // indirect
+	github.com/tetratelabs/wazero v1.8.2-0.20241030035603-dc08732e57d5 // indirect
```

Comment:
I think this can be updated to a released version now.

https://github.com/tetratelabs/wazero/releases/tag/v1.8.2 is released on Nov 25, 2024